<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QWidget" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome,Friend!</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To help us learn more about you and your design, please fill in your personal information below. We promise to keep all your information secret. &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Of course you can just ignore it if it troubles you.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="userBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nickname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(no more than 10 characters)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="GendercomboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Secret</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="AgespinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Age:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Job:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="JobcomboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Secret</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Teacher</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Student</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Designer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Businessman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remarks:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="remarksEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(no more than 100 words)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
